fixtures/showtec/phantom-25-led-wash.json

```
@@ -0,0 +1,413 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Phantom 25 LED Wash",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Hannes Rüger"],
+    "createDate": "2021-07-20",
+    "lastModifyDate": "2021-07-20"
+  },
+  "links": {
+    "manual": [
+      "https://www.manualslib.com/manual/835871/Showtec-Phantom-25-Led-Wash.html"
+    ],
+    "productPage": [
+      "https://www.musikhaus-korn.de/de/showtec-phantom-25-led-wash-moving-head/pd/47262"
+    ]
+  },
+  "physical": {
+    "dimensions": [205, 230, 210],
+    "weight": 3.7,
+    "power": 105,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "25W White LED",
+      "lumens": 543
+    },
+    "lens": {
+      "degreesMinMax": [13, 13]
+    }
+  },
+  "availableChannels": {
+    "Pan": {
+      "capability": {
+        "type": "Pan",
+        "angleStart": "0deg",
+        "angleEnd": "540deg"
```

```suggestion
        "type": "Pan",
        "angleStart": "0deg",
        "angleEnd": "540deg",
        "fineChannelAliases": ["Pan fine"],
```
Fine Channel missing.